openshift/setup-minishift.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e -x
 
-minishift profile set openshift-datagrid-tutorials
+minishift profile set datagrid-quickstart
```

Comment:
@galderz I hope this name change is ok. `datagrid-quickstart` seems like an easier profile name to work with.